html/cross-origin-opener-policy/resources/javascript-url-same-origin.https.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<script src="/common/get-host-info.sub.js"></script>
+<script>
+'use strict';
+const params = new URLSearchParams(location.search);
+const childURL = JSON.stringify(encodeURIComponent(params.get('childURL')));
+const childName = JSON.stringify(params.get('childName'));
+location.assign(`javascript:\`<script>
+const child = window.open(${childURL}, ${childName});
+const bc = new BroadcastChannel(${childName});
+
+bc.addEventListener('message', ({data}) => {
+  if (data === 'close') {
+    close();
+    return;
+  }
+  if (data === 'inspectChild') {
+    const iframe = document.createElement('iframe');
+    iframe.onload = () => {
+      const payload = {
+        childClosed: child.closed
+      };
+      iframe.contentWindow.postMessage(payload, '*');
+    };
+    iframe.src = '${get_host_info().HTTPS_ORIGIN}/html/cross-origin-opener-policy/resources/postback.html?channel=${params.get('childName')}';
+    document.body.appendChild(iframe);
```

Comment:
The reason the stability check fails in Firefox is because this is racy with the HTML parser. The `message` event handler here can be called before the `body` element exists! I'll submit a fix.